src/posit/connect/content.py

```diff
@@ -189,10 +189,18 @@ def dashboard_url(self) -> str:
     def app_role(self) -> str:
         return self.get("app_role")  # type: ignore
 
+    @property
+    def owner(self) -> dict | None:
+        return self.get("owner")
```

Comment:
Since `GET v1/content/{guid}/` does not allow you to include owner or tags, this should first check for the existence of `owner` in the data, and if not found, `GET` the `owner_guid`. 

Should this return a `User` instead of `dict`?

It should never return `None`.

Reply:
Should this return a User instead of dict?

The information here is a subset of fields defined in `User`. I considered defining an `Owner(Resource)` instead.

Reply:
> Should this return a User instead of dict?

I don't know, that was my question! 🤪 

`dict` is probably fine until we determine we need something richer (and then `User` is just a subclass of `dict` so it shouldn't be a breaking change if we were to upgrade it). The `dict` does include the `guid` so you could fetch the whole `User` if you wanted.

Reply:
lol, I forgot to quote it... 🤦🏻